test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-purchase_return@git+https://github.com/OCA/purchase-workflow.git@refs/pull/1975/head#subdirectory=setup/purchase_return
```

Comment:
Aborting merge, @JordiBForgeFlow could you remove this?

Reply:
@LoisRForgeFlow Done!